drivers/interrupt_controller/intc_miwu.c

```diff
@@ -59,6 +59,22 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(intc_miwu, LOG_LEVEL_ERR);
 
+/* Macro functions for MIWU multi-registers */
+#define NPCX_WKEDG(base, group) \
+	(*(volatile uint8_t *)(base +  npcx_wkedg_offset(group)))
+#define NPCX_WKAEDG(base, group) \
+	(*(volatile uint8_t *)(base + npcx_wkaedg_offset(group)))
+#define NPCX_WKPND(base, group) \
+	(*(volatile uint8_t *)(base + npcx_wkpnd_offset(group)))
+#define NPCX_WKPCL(base, group) \
+	(*(volatile uint8_t *)(base + npcx_wkpcl_offset(group)))
+#define NPCX_WKEN(base, group) \
+	(*(volatile uint8_t *)(base + npcx_wken_offset(group)))
+#define NPCX_WKINEN(base, group) \
+	(*(volatile uint8_t *)(base + npcx_wkinen_offset(group)))
+#define NPCX_WKMOD(base, group) \
+	(*(volatile uint8_t *)(base + npcx_wkmod_offset(group)))
+
```

Comment:
Can these go back to the reg_def.h? It feels like this is the common way to set these registers and I'm not sure there's a benefit to removing these.

Reply:
Done.